tests/drivers/pwm/pwm_loopback/testcase.yaml

```diff
@@ -6,3 +6,12 @@ tests:
     harness: ztest
     harness_config:
       fixture: pwm_loopback
+  drivers.pwm.esp32.loopback:
+    tags: pwm drivers userspace
+    depends_on: pwm
+    filter: dt_compat_enabled("test-pwm-loopback")
+    harness: ztest
+    harness_config:
+      fixture: pwm_loopback
+    platform_allow: esp32
+    extra_args: "CONFIG_MCPWM_ESP32=y"
```

Comment:
not required if enabling based on dt compat, see previous comment

Reply:
OK